src/main/java/com/easypost/model/CarrierAccount.java

```diff
@@ -241,6 +223,24 @@ public CarrierAccount update(final Map<String, Object> params, final String apiK
         return this;
     }
 
+    /**
```

Comment:
The ID getter/setter isn't in the right place? With the rules changes, shouldn't Checkstyle/your IDE complain about this?

Reply:
Odd, I'll check it out.

Reply:
I'm not sure. There's getter/setters for `id` in the base `EasyPostResource` class, but this seems to be the only one that gets tripped up by the override method.

NOTE FOR FUTURE: We shouldn't be overriding the getter/setters for `id` for no real reason...